ante/fee_unit_test.go

```diff
@@ -0,0 +1,39 @@
+package ante
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	bank "github.com/cosmos/cosmos-sdk/x/bank/types"
+	evidence "github.com/cosmos/cosmos-sdk/x/evidence/types"
+	"github.com/stretchr/testify/assert"
+
+	leverage "github.com/umee-network/umee/v3/x/leverage/types"
+)
+
+func TestPriority(t *testing.T) {
+	tcs := []struct {
+		name     string
+		oracle   bool
+		msgs     []sdk.Msg
+		priority int64
+	}{
+		{"empty priority 0", false, []sdk.Msg{}, 0},
+		{"when oracleOrGravity is set, then tx is max", true, []sdk.Msg{}, 100},
+		// dirty optimization - isOracleOrGravity flag takes a precendence
```

Comment:
Can we move this comment to the isOracleOrGravity godoc?

Reply:
OK, done